app/playground/page.tsx

```
@@ -1,43 +1,186 @@
+              <div>
+                <h3 className="text-2xl font-semibold text-foreground mb-4">Instant Feedback</h3>
+                <p className="text-muted-foreground mb-6">Get immediate results as you code, with syntax highlighting and error detection.</p>
+                <ul className="space-y-2 text-left">
+                  <li className="flex items-center gap-2 text-primary"><Zap className="w-4 h-4" /> Supports TEAL and PyTeal</li>
+                  <li className="flex items-center gap-2 text-primary"><Zap className="w-4 h-4" /> Simulate transactions</li>
+                  <li className="flex items-center gap-2 text-primary"><Zap className="w-4 h-4" /> Visual debugging tools</li>
+                </ul>
+              </div>
+              <div className="text-center">
+                <div className="bg-muted rounded-lg p-6 mx-auto max-w-sm">
+                  <pre className="text-sm font-mono text-foreground">/* Example TEAL Code */
```

Biome (2.1.2)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)